addons/website/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,64 @@
+                    <h2>Explore our<br/> key statistics</h2>
+                    <p><br/></p>
+                    <p class="lead">Analyzing the numbers behind our success: an in-depth look at the key metrics driving our company's achievements.</p>
+                    <p><a href="#" class="btn btn-primary" role="button">More details</a></p>
+                </div>
+                <div class="o_grid_item g-height-4 g-col-lg-4 col-lg-4 justify-content-start o_cc o_cc3" style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 5 / 5 / 9; z-index: 2;">
+                    <h3 class="display-3">12k</h3>
+                    <p><br/></p>
+                    <h4>Useful options</h4>
+                    <p>Explore a vast array of practical and beneficial choices.</p>
+                </div>
+                <div class="o_grid_item g-height-4 g-col-lg-4 col-lg-4 justify-content-start o_cc o_cc5" style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 9 / 5 / 13; z-index: 3;">
+                    <h3 class="display-3">45%</h3>
+                    <p><br/></p>
+                    <h4>More leads</h4>
+                    <p>Boost your pipeline with an increase in potential leads.</p>
+                </div>
+                <div class="o_grid_item g-height-4 g-col-lg-8 col-lg-8 justify-content-start o_cc o_cc4" style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 5 / 5 / 9 / 13; z-index: 4;">
+                    <h3 class="display-3">8+</h3>
+                    <p><br/></p>
+                    <h4>Amazing pages</h4>
+                    <p>Discover outstanding and highly engaging web pages.</p>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<!-- Options -->
+<template id="s_numbers_showcase_options" inherit_id="website.snippet_options">
+    <!--  Box Vertical Alignment -->
+    <xpath expr="//div[@data-js='WebsiteAnimate']" position="after">
+        <div data-selector=".s_numbers_showcase .o_grid_item" data-exclude=".o_grid_item_image">
+            <we-button-group string="Vert. Alignment" title="Vertical Alignment">
+                <we-button title="Align Top"
+                        data-select-class="justify-content-start"
+                        data-img="/website/static/src/img/snippets_options/align_top.svg"/>
+                <we-button title="Align Middle"
+                        data-select-class="justify-content-center"
+                        data-img="/website/static/src/img/snippets_options/align_middle.svg"/>
+                <we-button title="Align Bottom"
+                        data-select-class="justify-content-end"
+                        data-img="/website/static/src/img/snippets_options/align_bottom.svg"/>
+            </we-button-group>
+        </div>
+    </xpath>
+</template>
```

Comment:
Some comments about this option:
- 1/ Is it really necessary to have this option ? Given the base template, the `justify-content-start` is not really needed and I don't think it will be an issue if the user cannot align the content to the end for example :thinking: 

  If it is not necessary, then
    - Remove this option here
    - Remove the SCSS file (as it was used to have `display: flex` so this option can work) as well as its "assets definition" below 
    - Remove the `justify-content-*` classes from the grid items

- 2/ If you think it is really needed:
  - Replace the button icons by their "solo" version (like in Masonry), because here, they are the icons when aligning columns between them, so we need the ones showing the alignment inside a column :wink: 
  - The option is targeting the `.o_grid_item` elements, but when we toggle the "Cols" mode (I call it normal mode) the option is still there, and it therefore has no effect since the column is not `display: flex` anymore :+1: Therefore, you have three options:
 
    a. Add an `options.js` file, with an override of the `_computeWidgetVisibility` function in it, which will hide the widget/option depending on some conditions (so here if the grid mode is activated or not) Note that you will need to put the alignment option in a `data-js` div :slightly_smiling_face: 

    b. Allowing the option for both grid and normal modes => every place you used `.o_grid_item` will need to be replaced by `.row > div` so it targets columns (so in the CSS and here in the option) :+1: But then, what about the images, since you excluded them ?

    c. Do like masonry and make this snippet grid-only: so in the `snippets.xml` file, exclude the snippet from the "Grid and columns" part and add it in the selector of the "Grid only" part :slightly_smiling_face: 

  I personally think options `a` and `b` are the best, since `c` would remove some design freedom :thinking: And between `a` and `b`, depending on if you find what to do for the image columns you excluded, then `b` could be the easiest or not :grimacing: I will let you choose :+1: 

I personally would just remove this option, given the timing, only masonry needed it until now after all and it was easy for it since it is grid-only :shrug: 
If we notice that more an more snippets would need it (which is apparently the case), then we (website) will just make a more general option for grid items in general, instead of copying this code from a snippet to another :slightly_smiling_face: (to confirm)

Reply:
It was to customize some themes, but I guess it's fine to remove it 👍 

> If we notice that more an more snippets would need it (which is apparently the case), then we (website) will just make a more general option for grid items in general

It will certainly be necessary, other new snippets will use it :)